tests/shared/src/test/scala/scala/meta/tests/parsers/XmlSuite.scala

```diff
@@ -4,6 +4,7 @@ import scala.meta._
 import scala.meta.dialects.Scala211
 
 import org.scalameta.logger
+import munit.TestOptions
```

Comment:
nothing in the file changed. do we need the import?

Reply:
That's leftover, will fix it!

Reply:
Removed